scss/_progress.scss

```diff
@@ -30,9 +31,6 @@
 }
 
 .progress-bar {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
```

Comment:
_Only tested on Firefox / macOS_

There is a small regression regarding the vertical alignment of the text in the bar compared to the _main_ branch. The text seems too close to the bottom.

For the LTR mode, it is just a bit off:

![Screenshot 2022-11-11 at 10 34 18](https://user-images.githubusercontent.com/17381666/201311576-90ec4bdd-e9b6-43c8-aa88-3b70dfae868e.png)

For the RTL mode, it is now very close to the bottom:


![Screenshot 2022-11-11 at 10 34 24](https://user-images.githubusercontent.com/17381666/201311667-2eee0fe8-2f89-44c4-bbc6-46f52e36bc87.png)

Reply:
ah, good catch. i ripped those styles out because it looked like they were there mainly to make the old multi-bars work somehow. i'll review more thoroughly this weekend. we also need to check that it's indeed non-breaking i think

Reply:
Fixed...reintroduced those styles, and all seems fine